grand_challenge_forge/partials/example-evaluation-method/example-evaluation-method{{cookiecutter._}}/evaluate.py.j2

```diff
@@ -124,6 +98,30 @@ def process(job):
         "my_metric": random.choice([1, 0]),
     }
 
+def main():
```

Comment:
Pushed the process function up, as to make it the first thing that a reader sees.